src/reanimated2/UpdateProps.js

```diff
@@ -27,23 +27,33 @@ const ColorProperties = makeShareable(colorProps);
 export default function updateProps(viewDescriptor, updates, maybeViewRef) {
   'worklet';
 
-  const viewName = viewDescriptor.value.name || 'RCTView';
+  const updateSingleProps = (_viewDescriptorValue, _updates, _maybeViewRef) => {
+    const viewName = _viewDescriptorValue.name || 'RCTView';
 
-  if (Platform.OS !== 'web') {
-    Object.keys(updates).forEach((key) => {
-      if (ColorProperties.indexOf(key) !== -1) {
-        updates[key] = processColor(updates[key]);
-      }
-    });
-  }
+    if (Platform.OS !== 'web') {
+      Object.keys(_updates).forEach((key) => {
+        if (ColorProperties.indexOf(key) !== -1) {
```

Comment:
```suggestion
        if (ColorProperties.includes(key)) {
```

Reply:
I'm not sure why but when I changed `indexOf` to `includes` I received exception: 
![image](https://user-images.githubusercontent.com/36106620/102621659-2d615d00-4140-11eb-8ea0-3cf6517086db.png)

Reply:
This is very weird, should work OOTB. I searched for usages of `indexOf` in case it's treated special and found none. There's nothing in Hermes issues either.

Maybe @karol-bisztyga or @Szymon20000 know why this doesn't work?

Reply:
@jakub-gonet @piaskowyk shouldn't `includes` be also included in [the blacklist array](https://github.com/software-mansion/react-native-reanimated/blob/aa315669402dd3bcdf882197540275d075eef12c/plugin.js#L72-L123)?

Reply:
Must've missed that. Yeah, adding this to the blacklist should work.